Source/Eto.WinForms/WinConversions.cs

```diff
@@ -403,7 +403,7 @@ public static MouseButtons ToEto(this swf.MouseButtons button)
 
 		public static Graphics ToEto(this sd.Graphics g)
 		{
-			return new Graphics(new GraphicsHandler(g));
+			return new Graphics(new GraphicsHandler(g, false));
```

Comment:
Right, but we do not want to dispose when the Paint event is called.. e.g. this is incorrect winforms code:

```c#
var ctl = new Control();
ctl.Paint += (sender ,e) => {
  // do painting

  e.Graphics.Dispose();
};
```

with this change, this is effectively what will happen which will probably cause issues.